.stylelintrc

```diff
@@ -22,6 +22,7 @@
     "declaration-colon-space-after": "always",
     "declaration-colon-space-before": "never",
     "order/properties-alphabetical-order": true,
-    "length-zero-no-unit": true
+    "length-zero-no-unit": true,
+    "color-hex-length": "short"
```

Comment:
Add this in sorted order (Makes it easier to maintain [find/edit].)

Reply:
@DubeySandeep yeah sure nice suggestion.

Reply:
@DubeySandeep Done!